.github/sync-repo-settings.yaml

```diff
@@ -5,9 +5,6 @@ branchProtectionRules:
 - pattern: main
   isAdminEnforced: true
   requiredStatusCheckContexts:
-    - 'PHP 8.0 Unit Test'
-    - 'PHP 8.0 Unit Test protobuf,grpc'
-    - 'PHP 8.0 Unit Test --prefer-lowest'
```

Comment:
We need to increment the `protobuf,grpc` and `--prefer-lowest` tests to `PHP 8.1` instead of just removing them.

Reply:
Added in [44be7ac](https://github.com/googleapis/gax-php/pull/598/commits/44be7ac778e6e5fb4f776bc18b1f386b7affbf2c)